modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-impl/src/main/java/com/liferay/headless/commerce/admin/order/internal/util/v1_0/OrderItemUtil.java

```diff
@@ -102,6 +102,7 @@ public static CommerceOrderItem addCommerceOrderItem(
 					commerceOrder.getCommerceOrderId(),
 					cpInstance.getCPInstanceId(),
 					GetterUtil.getString(orderItem.getUnitOfMeasure()),
+					GetterUtil.getString(orderItem.getOptions(), null),
```

Comment:
I think `, null` is not needed as by default the `GetterUtil.getString` as a blank string (that you should catch with the `Validator.isBlank` you added). 
If instead you want `null` (so not a blank string) you can avoid the usage of `GetterUtil.getString` as `orderItem.getOptions()` already returns null if not provided